handleData.js

```diff
@@ -119,11 +169,43 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = {};
+    this.money = { "Copper"      : 0,
```

Comment:
Yeah I definitely had some kind of brain fart there. Fix will be in the next commit :+1: